src/inverted-index.js

```diff
@@ -1,31 +1,34 @@
+/**
+ * Inverted index class
+ */
 class InvertedIndex {
-
+  /**
+   * Inverted index constructor
+   */
   constructor() {
-    //Object to hold the index
+    // Object to hold the index
     this.index = {};
   }
 
-  /** 
-   * @param{String} words - String to be tokonized
-   * @return{Array} cleanContent
+  /**
+   * @param{String} words - String to tokenize
+   * @return{Array} list of words devoid of special characters or symbols
    */
-  tokenize(words) {
-    const cleanContent = words.trim().replace(/-/g, ' ')
+  static tokenize(words) {
+    return words.trim().replace(/-/g, ' ')
       .replace(/[.,\/#!$%\^&@\*;:'{}=\_`~()]/g, '')
-      .toLowerCase().split(' ').sort();
-    return cleanContent;
+      .toLowerCase()
+      .split(' ')
+      .sort();
   }
 
   /**
    * @param{String} words - The string to be filtered
    * @return{Array} tokens - Without duplicated words
    */
   uniqueWords(words) {
```

Comment:
You've declared this function as an instance method but you're using it like you would a class method.
If you intend to use it as a class method you should declare it as such.

Check out `static` methods.